server/images.go

```diff
@@ -1186,6 +1218,16 @@ func makeRequestWithRetry(ctx context.Context, method string, requestURL *url.UR
 		}
 	}
 
+	if anonymous {
+		// no user is associated with the public key, and the request requires non-anonymous access
+		pubKey, nestedErr := auth.GetPublicKey()
```

Comment:
Since this is only be used for comparison purposes, it would be better to use the fingerprint rather than the key itself

Reply:
I am going to leave it as the full key for now, because although the fingerprint comparison looks better in code, it makes the error message returned from the API less useful for users in my opinion.

Current error:
```
Error: unauthorized: unknown ollama key "ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIAPqCbrQdButwfawlgGiY7RUhOFAwPdKpN2QOZZ8zYRF"
```
This tells you what you need to add to ollama.com, and can be easily compared by a human to what is on their system.

Fingerprint error:
```
Error: unauthorized: unknown ollama key fingerprint "00:11:22:33:44:55:66:77:88:99:aa:bb:cc:dd:ee:ff"
```

Reply:
If we supported rsa keys, I think the fingerprint _might_ be nicer, but since we're only using ed25519 keys, showing the entire pub key is a lot cleaner. How would someone even know where to get the fingerprint?

Reply:
I thought the goal of setting this server side is to report the server key, i.e. the key used for push, and only show the message if the user has access to the key to import? In that case, the server doesn't need to send the key itself but the fingerprint since it's only used for comparison. The error still display the real key but only after verifying it's matches the one user has access to